spec/lib/configuration_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+module FloodRiskEngine
+  describe Configuration do
+    it "adds #config to the engine module" do
+      expect(FloodRiskEngine).to respond_to(:config)
+    end
+
+    it "has the correct default values" do
+      expect(FloodRiskEngine.config.test_me).to eq "test"
+    end
```

Comment:
@reggieb I should have made it clear in the code that test_me was designed to be replaced by the first real config value and was just there to set a precedent - but that was not a poor idea so have made it an example as you suggest. Now test for non-existence as you suggest too - thx